beacon_node/client/src/builder.rs

```diff
@@ -199,6 +206,16 @@ where
             builder
         };
 
+        let builder = if config.network.enable_light_client_server {
+            let (tx, rv) = tokio::sync::mpsc::channel::<LightclientProducerEvent<TEthSpec>>(
```

Comment:
I don't know if there's been any discussion on whether we use `tokio::sync::mpsc` vs `futures::channel::mpsc`, and it seems like both are currently used in Lighthouse. I don't know if we prefer one over another, It might be worth considering sticking with `futures::channel::mpsc` for now since it's the one used in `beacon_chain` to avoid using both types in the same file? (cc @michaelsproul)

Reply:
Just reading the docs and it looks like the `sender.try_send` for both crates behaves similarly - returns immediately and will return error if the buffer is full, which is what we want I think.

Reply:
Was debating both options at the time, happy to go with futures for consistency